qucs/translations/Makefile.am

```diff
@@ -75,7 +77,7 @@ qucs_.ts: $(qucs_SOURCES)
 	  -locations relative -ts $(srcdir)/$$file; \
 	done
 
-.ts.qm:
+%.qm: %.ts
```

Comment:
Is this portable?
http://stackoverflow.com/questions/767922/portable-equivalent-of-gnu-make-style-pattern-rules
I recall someone using FreeBSD comented about it... perhaps not a big issue, just need to use gmake.